skia-bindings/build_support/skia_bindgen.rs

```diff
@@ -481,6 +481,8 @@ const BLOCKLISTED_TYPES: &[&str] = &[
     "SkRuntimeEffect_ConstIterable.*",
     // Linux LLVM9 c++17
     "std::_Rb_tree.*",
+    // archlinux
```

Comment:
Just putting it out there that there's a good chance this will affect other distros too in the future, Archlinux is likely just ahead of the curve. I'm on clang/llvm 20.1.7 if that says anything.